tests/data/test_federated_dataset.py

```diff
@@ -266,7 +266,7 @@ def test_3_workers(self, mock_get_ops, federated_dataset_mixture,
 @pytest.fixture
 def user_id_to_weight(request):
     if hasattr(request, 'param') and request.param:
-        return {i: i for i in range(100)}
+        return {i: i + 1 for i in range(100)}
```

Comment:
I presume the "1" is for base weight? Perhaps a comment, or creating a variable name, `base_weight=1`, would help with legibility.

Reply:
this is just to avoid a user with weight 0